data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -24,13 +24,33 @@ function signal_bootstrap_complete {
   done
 }
 
+function release_cvo_lease {
+  if [ ! -f /opt/openshift/release_cvo_lease.done ]
+  then
+    until [ "$(oc get leases -n openshift-cluster-version version | grep -c "version")" -eq 0 ];
```

Comment:
I don't understand the need to loop here.  By the time we're at this stage, can we just run a single-shot deletion attempt?

Reply:
The loop is necessary for cases where the delete command fails, when it just works a single-shot delete would work, in case it fails this loop will check if the lease exists and keep trying to delete it.